.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
 matrix:
   fast_finish: true
   allow_failures:
-    - go: 1.14.3
+    - go: tip
```

Comment:
why test against go tip for this one?

Reply:
Reverted. I was investigating whether version of the go impacts race-detection flakiness.